README.md

```diff
@@ -222,6 +223,187 @@ There are number of techniques you can use to ensure that access token expiry is
 - Retain the access token along with the timestamp of when it was requested so you can verify ahead of time whether the token has already expired based on the `time-to-live` being used by your server.
 - Prefetch the access token whenever the `UIApplication`, or `UIViewController` associated with an outgoing call is created.
 
+## Migration Guide

```

Comment:
did we have stats of anything similar to stats in 2.x ? should migration guide include `stats` ? @idelgado ?

Reply:
Nope stats is not available on 2.x. Might not be a bad idea.

Reply:
Yup. Looks like I missed that. In retrospect the migration guide could have excluded new features and become it's own section altogether. For now we can put everything in the migration guide but long-term we can have 2 sections: `Migration Guide` and `New Features`.